airbrake/notifier.py

```diff
@@ -96,6 +97,8 @@ def __init__(self, project_id=None, api_key=None, host=None, timeout=None,
         self.blacklist_keys = config.get("blacklist_keys", [])
```

Comment:
We do have regex filters in the ruby notifier, and we should definitely add them. I've added an issue for that here: https://github.com/airbrake/airbrake-python/issues/69.

I think some other sdks have default blacklisted keys, and ya we should probably add some like `username`, and `password`. I'll go and check out what consensus is for other systems.